Mathlib/Analysis/InnerProductSpace/Symmetric.lean

```diff
@@ -214,6 +214,13 @@ theorem IsSymmetric.inner_map_self_eq_zero {T : E →ₗ[𝕜] E} (hT : T.IsSymm
   simp_rw [h _]
   ring
 
+theorem ker_le_ker_of_range {A B : E →ₗ[𝕜] E} (hA : A.IsSymmetric) (hB : B.IsSymmetric)
+    (h : range A ≤ range B) : ker B ≤ ker A := by
```

Comment:
This is a bit of a nit, but since the rest of the file uses `S` and `T`, can you use that instead of `A` and `B`?